myplugin/content/example/templates/example/fragments/use_case_template.html

```diff
@@ -0,0 +1,199 @@
+                    </li>
+                    <li>Step 2: Optionally enter a minimum volume size. This is for extra storage in the vm.
+                        For this example, set the volume size to {{ example_info.preset.volume_size_gb }} GB.
+                    </li>
+                </ul>
+            </li>
+            <li><strong>Configure Security Groups</strong>
+                <ul>
+                    <li>Select one or more existing security groups to be used for the
+                        AppTemplate. These security groups define the network access rules for the
+                        instances launched from this AppTemplate.
+                        {{ example_info.security_group_instructions }}
+                    </li>
+                </ul>
+            </li>
+            <li><strong>Write Configuration Script:</strong>
+                <ul>
+                    <li>Step 1: Enter instantiation and account attributes to be used in the configuration
+                        script. Those are variables used in the script. They will be filled with values
+                        during the instantiation of the AppTemplate. Account Attributes can be used for user
+                        related data where each user has different data (e.g. username). Instantiation
+                        attributes are only filled once per Instance (e.g. java-version).
+                        {% if example_info.preset.instantiation_attributes %}
+                            In this case use
+                            {% for attr in example_info.preset.instantiation_attributes %}
+                                {% if not forloop.first %}:{% endif %}{{ attr.name }}
+                            {% endfor %}
+                            as instantiation attributes.
+                        {% else %}
+                            In this AppTemplate Example you do not need instantiation attributes.
+                        {% endif %}
+
+                        {% if example_info.preset.account_attributes %}
+                            In this case use
+                             {% for attr in example_info.preset.account_attributes %}
+                                {% if not forloop.first %}:{% endif %}{{ attr.name }}
+                            {% endfor %}
+                            as account attributes.
+                        {% else %}
+                            In this AppTemplate Example you do not need account attributes.
+                        {% endif %}
+                    </li>
+                    <li>Step 2: Click the "Write Script" button to open the script editor.</li>
+                    <li>Step 3: Write the following cloud-init configuration script to automate the setup of
+                        the {{ example_info.use_case_title }}:
+                    </li>
+                    <pre><code>
+                        {{ example_info.preset.script|safe }}
+                    </code></pre>
```

Comment:
```suggestion
                    <pre><code>{{ example_info.preset.script|safe }}</code></pre>
```

`<pre>` und `<code>` müssen direkt vor der ersten Codezeile und direkt an/hinter der letzten sein, damit das formatting richtig funktioniert und angezeigt wird